db/fixtures/tactical_vote_stt_recs.rb

```diff
@@ -5,9 +5,10 @@
 require_relative("mysociety_constituencies_csv")
 
 class TacticalVoteSttRecs
-  ACCEPTABLE_NON_PARTY_ADVICE = ["None"]
   attr_reader :advisor, :mysoc_constituencies
 
+  ACCEPTABLE_NON_PARTY_ADVICE = ["Heart"]
```

Comment:
IIUC this doesn't make much sense to users, shouldn't we have some human-readable text which explains what it means?

Reply:
I think the only options available to us are:

* change the link back to STT to a page that explains what this means (which doesn't exist yet).
* make an adjustment to the translation of the text "heart" on import.

BUT, the second option, amounts to us interpreting what STT have said (unless of course, we're just keeping up with their UI). Which is questionable, given the neutral positioning of SMV.

Reply:
I’m happy with doing what amounts to a precis, just not substantively changing or interpreting the meaning. We can always link out for further context, and in fact should be doing this somehow (without driving our users away of course)

Reply:
What STT say is

> Where we think Tories & RefUK can't win "Heart"

and that's been explained to me as "To stop the Tories, you can safely vote with your heart, because there won't be an impact"

Another way of saying that is "Any"

**PROPOSAL**: for the "heart" constituencies, we display the text "Any" (instead of party name)

Reply:
"Any" makes a ton more sense than "Heart" so yeah let's do that for now. However I'm convinced we can make it clearer with follow-up PRs to add a tooltip or similar which links out to the relevant page for an explanation.